File: Know Basic maths/python/rev_num.py
def rev(num:int) -> int:
    rev = 0
    if num == 1 or num == 0: print("Enter the number again")

    while num > 0:
        last_dig = num % 10
        rev = rev*10 + last_dig
        num = num // 10
    
    return rev

n = int(input("Enter the number: "))
print("is palindrone:" , n == rev(n))

File: Know Basic maths/C++/rev_num.cpp
#include <iostream> 
using namespace std;

int rev(int n){
    int rev = 0;
    float last_dig;
    while (n > 0){
      last_dig = n % 10;
      rev = rev*10 + last_dig;
      n = n / 10;
    }
    return rev;
}

int main(){
    int N;
    cout << "Enter the number : " ;
    cin >> N;
    int rev_digit = rev(N);
    cout << "The reverse Digits is " << rev_digit << endl;
} 